src/navigators/LoggedInStack.tsx

```diff
@@ -80,6 +81,10 @@ export function LoggedInStack() {
 
   const loadingMessage = getLoadingMessage();
 
+  useEffect(() => {
+    activeSubjectId && _sdkAnalyticsEvent.setUser(activeSubjectId);
```

Comment:
I would think we would want to make the user be the userId and set the subjectId as a property.